stratum/tools/stratum-replay/README.md

```diff
@@ -0,0 +1,28 @@
+<!--
+Copyright 2020-present Open Networking Foundation
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+P4Runtime write request replay tool
```

Comment:
It would be good to include tutorial-style instructions, e.g. where to find the write log and pipeline config and p4info file in a running Stratum container, how to extract files from the container, finally example invocation of `stratum-replay` with the obtained files.